salt/metalk8s/solutions/configured.sls

```diff
@@ -0,0 +1,18 @@
+{%- set solutions = pillar.metalk8s.solutions.unconfigured %}
```

Comment:
For me this state should include the mounted one (or the init.sls should include both) and you don't need an orchestrate for mounted IMO

Reply:
yes i guess one init.sls will be enough